aiogram/bot/bot.py

```diff
@@ -176,6 +177,43 @@ async def get_me(self) -> types.User:
         result = await self.request(api.Methods.GET_ME, payload)
         return types.User(**result)
 
+    async def log_out(self) -> base.Boolean:
+        """
+        Use this method to log out from the cloud Bot API server before launching
+        the bot locally. You **must** log out the bot before running it locally,
+        otherwise there is no guarantee that the bot will receive updates.
+        After a successful call, you will not be able to log in again using the
+        same token for 10 minutes. Returns True on success. Requires no parameters.
+
+        Source: https://core.telegram.org/bots/api#logout
+
+        :return: Returns True on success
+        :rtype: :obj:`base.Boolean`
+        """
+        payload = generate_payload(**locals())
+
+        result = await self.request(api.Methods.LOG_OUT, payload)
+        return result
+
+    @deprecated("This method will be renamed to `close` in aiogram v3.0")
+    async def close_bot(self) -> base.Boolean:
+        """
+        Use this method to close the bot instance before moving it from one local
+        server to another. You need to delete the webhook before calling this method
+        to ensure that the bot isn't launched again after server restart. The method
+        will return error 429 in the first 10 minutes after the bot is launched.
+        Returns True on success. Requires no parameters.
+
+        Source: https://core.telegram.org/bots/api#close
+
+        :return: Returns True on success
+        :rtype: :obj:`base.Boolean`
+        """
+        payload = generate_payload(**locals())
+
+        result = await self.request(api.Methods.CLOSE, payload)
```

Comment:
let's just return call result, without creating new name.

```python3
     return await self.request(api.Methods.CLOSE, payload)
```

same for .log_out

Reply:
It's a code style of all bool return method (look at other methods). 
I agree that it's need to be refactored, but **all** of them :)